### ISSUE TITLE
Revise Checklists

### ISSUE THREAD

**Issue by Ralphy444** [Ralphy444/Infinite-Checklist#8]
Revise checklists to ensure accuracy and add or remove things accordingly.

**Comment by rohaimkhedr** [comment]
i will do this

**Comment by rohaimkhedr** [comment]
on the taxi checklist you can remove the following things
Engines running (they are always running)
by takeoff list you can remove 
strobes on you must  do the lights by taxi

**Comment by Ralphy444** [comment]
Nope, this app was built with Global in mind so the engine's running should be there. Also, negative, landing and strobes should be put on as you enter the runway not when you taxi.

**Comment by rohaimkhedr** [comment]
Then this issue is done
